compiler/control/OMROptions.cpp

```diff
@@ -1359,6 +1359,13 @@ OMR::Options::set32BitNumeric(char *option, void *base, TR::OptionTable *entry)
 
 
 char *
+OMR::Options::set32BitNumericInJitConfig(char *option, void *base, TR::OptionTable *entry)
```

Comment:
This is a new function.  What is the purpose of the `base` parameter because its unused here and in all implementations in this PR?  Please add a Doxygen-style comment (perhaps in the header file where the other comment is already).